channels/sa.m3u

```diff
@@ -1,10 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="22TV.sa" tvg-name="22 TV" tvg-country="SA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/m5cc4g5.png" group-title="General",22 TV
 http://82.212.74.99:8000/live/hls/8117.m3u8
-#EXTINF:-1 tvg-id="AfricaTV2.sa" tvg-name="Africa TV2" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="",Africa TV2 (720p)
-http://africatv.live.net.sa:1935/live/africatv2/playlist.m3u
```

Comment:
There's one channel that exists called http://africatv.live.net.sa:1935/live/africatv/playlist.m3u8

I think the others are Not 24/7.

Reply:
The link works it's just whoever put it in forgot the "8" in .m3u8